### DISCUSSION TITLE
Maplibre support

### DISCUSSION THREAD

**Discussion by arcoirislabs** [DioxusLabs/dioxus#2260] [General]
## Feature Request

How to render maps done by maplibre library which are possible in Webview based apps??

**Comment by kylerchin** [comment]
+1, really curious about this as well

**Comment by ydrea** [comment]
no one seems to be working on maps... would anyone be willing to form a team?

**Comment by crackedbeefcake** [comment]
did you guys figure anything out here? i am also trying to render a map. for a small mac native app.

**Reply by mickeys000** [reply]
> did you guys figure anything out here? i am also trying to render a map. for a small mac native app.

Someone did [here](https://github.com/lmmx/dx-map)

And, also, development is not superfast, but MapLibre itself seems to be working on [something](https://github.com/maplibre/maplibre-native-rs).

**Comment by stevenvergenz** [comment]
Sorry to resurrect an old thread, but there is an attempt to make dx bindings for Leaflet here: https://github.com/lheintzmann1/dioxus-leaflet

**Reply by mickeys000** [reply]
As I mentioned [above](https://github.com/DioxusLabs/dioxus/discussions/2260#discussioncomment-15147990), there's some work in the MapLibre world too, although towards Rust in general, rather than Dioxus specifically.
